packages/components-js/src/pages/pin-code.html

```diff
@@ -6,6 +6,10 @@
   <p-pin-code label="Some Label" description="Some Description"></p-pin-code>
 </div>
 
+<div class="playground light" title="should render with label, description and length=1">
+  <p-pin-code label="Some Label" description="Some Description" length="1"></p-pin-code>
```

Comment:
Do we need visual coverage for the other lenghts, too? Seems like some are missing now.